rbd/features.go

```diff
@@ -91,6 +91,43 @@ const (
 	RbdFeaturesSingleClient   = uint64(C.RBD_FEATURES_SINGLE_CLIENT)
 )
 
+// FeatureSet is a combination of the bit value for multiple featurs.
+type FeatureSet uint64
+
+var (
+	// This isn't a const, but it should be. A map can not be const?
```

Comment:
No it can't be. There are various discussions of that in go language forums and a few language enhancement proposals floating around. We probably don't need to mention it here, unless you just want to say `// treat as constants after init` or something along those lines.